### PR TITLE
feat: standalone client supports to update catalog

### LINKED ISSUE #461: Update catalog in stand alone mode

### PR DESCRIPTION
**Pull request by vagetablechicken**
Closes #461 
Nameserver stores sp infos. And client get sp infos from nameserver.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -71,7 +72,7 @@ java/openmldb-import/src/main/java/com/_4paradigm/openmldb/proto/
 java/openmldb-nearline-tablet/src/main/java/com/_4paradigm/openmldb/proto/
 java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/proto/
 python/logs
-python/sqlalchemy-test/*.xml
+/python/test/*.xml
```
Comment (dl239) [inline .gitignore:75]: ./python/test/*.xml  ?

Comment (vagetablechicken) [reply .gitignore:75]: `/` means the root dir

File: src/sdk/db_sdk.cc [hunk 2]
```diff
@@ -428,24 +425,38 @@ bool StandAloneSDK::BuildCatalog() {
...
+    std::vector<api::ProcedureInfo> procedures;
+    // empty db & sp names means show all
+    if (!GetNsClient()->ShowProcedure("", "", &procedures, &msg)) {
+        LOG(WARNING) << "show procedure from ns failed, msg: " << msg;
+        return false;
+    }
+    // api::ProcedureInfo to hybridse::sdk::ProcedureInfo
+    catalog::Procedures db_sp_map;
+    for (auto& sp : procedures) {
+        auto sdk_sp = openmldb::catalog::SchemaAdapter::ConvertProcedureInfo(sp);
```
Comment (dl239) [inline src/sdk/db_sdk.cc]: `ConvertProcedureInfo` has been deleted, please merge main branch

Comment (vagetablechicken) [reply src/sdk/db_sdk.cc]: ok

**Review by dl239 — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**